src/app/page.js

```diff
@@ -1,160 +0,0 @@
-import Card from "@/Components/Card";
-import Header from "@/Components/Header";
-import Navbar from "@/Components/Navbar";
-import SchoolView from "@/Components/SchoolView";
-import Slider from "@/Components/Slider";
-import bgBlue from "../../public/Assets/Home/bgBlue.jpg";
-import bgContact from "../../public/Assets/Home/slider1.jpg";
-import faq from "../../public/Assets/Home/faq.jpg";
-import { CARD_DATA, wyChooseUs } from "@/Data/CARD_DATA";
-import ScrollToTop from "@/Components/ScrollToTop";
-import Footer from "@/Components/Footer";
-import Image from "next/image";
-import Faq from "@/Components/Faq";
-
-
-export default function Home() {
-  const handleSubmit  = async (event) => {
-    event.preventDefault();
-    const formData = new FormData(event.target);
-
-    formData.append("access_key", "YOUR_ACCESS_KEY_HERE");
```

Comment:
Why is this handleSubmit the same as the one on the form page.js.

Please have a form component and then the handleSubmit logic on the page